_data/tools.csv

```diff
@@ -122,7 +122,8 @@ PythonFMU,PythonFMU,https://github.com/NTNU-IHB/PythonFMU/,A Python tool couplin
 RecurDyn,RecurDyn,https://eng.functionbay.co.kr,High End Multi Flexible Body Dynamics Software from FunctionBay,commercial,win64,available,available,planned,planned,available,available,planned,planned
 Romax Nexus,RomaxFMUExport,https://www.romaxtech.com/software/,"Romax Nexus, from Romax Technology, is a suite of CAE tools for the design, simulation and optimisation of transmissions and electromechanical power trains for durability, NVH and efficiency",commercial,linux32 linux64 win64 win32,,planned,,,,,,
 RTMaps,RTMaps,https://intempora.com/products/rtmaps/,Component-based software development and execution environment which enables users to time-stamp record synchronize and play back data from various sensors and vehicle buses,commercial,linux32 linux64 win32 win64,,,,,,available,,
-SaberRD,SaberRD,https://www.synopsys.com/saber,"SaberRD is an intuitive, integrated environment for designing and analyzing power electronic systems and multi-domain physical systems",commercial,linux32 linux64 win32 win64,,available,,,,,,available
+SaberEXP,SaberRD,https://www.synopsys.com/verification/virtual-prototyping/saber-exp.html,“SaberEXP is a piecewise linear (PWL) circuit simulator that enables design teams to start early and validate their power supply/converter and motor drive designs prior to physical implementation with its fast simulation convergence. It provides a scalable simulation solution for accelerating power electronics designs with its seamless export flow into SaberRD for high fidelity large system design”,commercial,linux64 win64,,available,,,,,,
```

Comment:
Tool id `SaberRD` is not unique (as reported by Circel CI). Should be `SaberEXP` for example.